cmd/tilde-config/main.go

```diff
@@ -41,10 +44,23 @@ func main() {
 	flag.StringVar(&settings.coastal, "coastal", "TG,LG", "coast tsunami gauge network code")
 	flag.StringVar(&settings.enviro, "enviro", "EN", "envirosensor network code")
 	flag.StringVar(&settings.manual, "manual", "MC", "manualcollect network code")
+	flag.StringVar(&settings.geomag, "geomag", "GM", "geomagnetic network code")
+	flag.StringVar(&settings.extra, "extra", "GM=SM_SMHS_50", "attach extra stations and locations to a network, e.g. GM=SM_SMHS_50")
 	flag.StringVar(&settings.output, "output", "", "output dart configuration file")
 
 	flag.Parse()
 
+	extra := make(map[string][]string)
+	for _, s := range strings.Split(settings.extra, ",") {
+		if s = strings.TrimSpace(s); s != "" {
+			keys := strings.Split(s, "=")
+			if len(keys) != 2 {
+				continue
```

Comment:
question: we don't want to throw error here? this is effectively passing incorrect args to the CLI?

Reply:
good point, yeah it's a bit transparent. will update.